builders/build/substrate-api/chopsticks.md

```diff
@@ -0,0 +1,167 @@
+---
+title: How to use Chopsticks to Fork Moonbeam
+description: Follow this tutorial to learn the basic of how to use Chopstick to locally fork the entirety of a Moonbeam network.
+---
+
+# How to Use Chopsticks to Fork Moonbeam
+
+![Chopsticks diagram](/images/builders/build/substrate-api/chopsticks/chopsticks-banner.png)
+
+## Introduction {: #introduction }
```

Comment:
I think adding the specific features of Chopsticks that are covered later on in the guide into the introduction will help provide a bit of context as to what you can use Chopsticks for, i.e., replaying blocks and for testing out XCM functionality